### ISSUE TITLE
SSH access to `test-macstadium-macos11.0-arm64-4`

### ISSUE THREAD

**Issue by UlisesGascon** [nodejs/build#3685]
I was able to connect to the machine using SSH, and then I ran the cleanup script. Everything was fine until I rebooted the machine. After the reboot, the SSH connection became broken. It seems like we have a similar issue as #3667.

I will add this to the support ticket opened for #3667.

**Comment by UlisesGascon** [comment]
This machine will be wiped and re-ansibled once https://github.com/nodejs/build/issues/3667 is working fine in order to avoid blocking Node.js 22.0.0 release due lack of macos-11-arm machines.

**Comment by targos** [comment]
Since both osx-11 arm machines are offline and currently impossible to access, I disabled node-test-commit-arm from https://ci.nodejs.org/job/node-test-commit/configure

**Comment by ryanaslett** [comment]
I put in a ticket with macstadium to have these rebooted and re-establish connectivity. 

We can now ssh into all 3 bare metal osx arm machines.

I had to wipe and re-ansible `test-macstadium-macos11.0-arm64-3` which failed to run a build.

Suspecting that something was awry with the updated homebrew packages, I went ahead and duplicated the /opt/homebrew/bin directory from `test-macstadium-macos11.0-arm64-4`, and re-ran a build, and it passed.

One other thing to note is that the builds seem to assume that ccache will be available at /usr/local/bin -> which is where homebrew installs on x86 machines. However, due to rosetta conflicts, apple silicon (arm) homebrew installs to /opt/homebrew/bin instead.

I symlinked /opt/homebrew/bin to /usr/local/bin, and the builds passed. 

Im going to go ahead and re-enable osx-11 node-test-commit-arm (node-test-commit-osx appears to be disabled as well?)

This should keep these machines afloat long enough for us to replace them with the new infra.
